packages/container/lib/container.js

```diff
@@ -379,31 +379,27 @@ function areInjectionsDynamic(injections) {
   return !!injections._dynamic;
 }
 
-function buildInjections() /* container, ...injections */{
+function buildInjections(container, ...injectionsArray) {
   let hash = {};
+  let injections = Array.prototype.concat(...injectionsArray);
```

Comment:
since `injectionsArray` is array of arrays, you still need concat step https://cl.ly/1L3w103u3116, it either had to be `[].concat(...injectionsArray)` or `Array.prototype.concat(...injectionsArray)`, perf-wise my assumption it would be better to concat all arrays inside `injectionsArray` immediately rather than `for loop`ing one by one.